docs/CHANGELOG.md

```diff
@@ -59,6 +59,10 @@ Secrets must be configured manually and passed via `-secrets <filename>` or equi
 
 - `NODE_NO_NEW_HEADS_THRESHOLD=0` no longer requires `NODE_SELECTION_MODE=RoundRobin`. 
 
+## 1.10.0 - 2022-11-15
+
+### Added
+
```

Comment:
I think that Bridge caching is new to 1.11
@essamhassan can you confirm?

Reply:
Ah, good catch. It does look like it's there. I'll move it.

Reply:
Updated in 0959f6e59e476c91405ef93688475f359ee5bd1c